configure.ac

```diff
@@ -365,6 +365,9 @@ AC_SUBST(PLATFORM_AOT_SUFFIX)
 
 ## PLATFORM_AOT_SUFFIX not so simple for windows :-)
 
+: ${CFLAGS="-g -O3"}
```

Comment:
Well, the whole point of doing it like this is so that the user can override it with `CFLAGS` in the environment. Can I just modify the `run-jenkins.sh` script in this PR?